### PR TITLE
Add views for aquilon clusters, services and GRNs

### PR DESCRIPTION
**Pull request by jrha**
As requested by @apdibbo.

### CODE REVIEW

File: views/view-aquilon-clusters.php
Comment (tofu-rocketry) [inline views/view-aquilon-clusters.php]: How's this work?

File: views/view-aquilon-grn.php
Comment (tofu-rocketry) [inline views/view-aquilon-grn.php]: This code looks familiar. :thinking:

File: views/view-aquilon-personalities.php
Comment (tofu-rocketry) [inline views/view-aquilon-personalities.php:47]: :+1:

**Comment by jrha** [comment]
Rebased.

**Comment by jrha** [comment]
Should be updated after #129 is merged.

**Comment by jrha** [comment]
@apdibbo @tofu-rocketry rebased

File: views/view-aquilon-services.php
Comment (tofu-rocketry) [inline views/view-aquilon-services.php]:
```suggestion
        }
```

File: views/view-aquilon-services.php
Comment (tofu-rocketry) [inline views/view-aquilon-services.php]:
```suggestion
        }
```

File: views/view-aquilon-services.php
Comment (tofu-rocketry) [inline views/view-aquilon-services.php:31]: From my old comments: how does this work?

Comment (jrha) [reply views/view-aquilon-services.php:31]: What do you mean? `file_get_contents` returns `false` if it fails and the triple equals operator in PHP checks type as well as value to ensure it isn't the string "false", because PHP.

Comment (tofu-rocketry) [reply views/view-aquilon-services.php:31]: Thanks. And then `error`?

Comment (jrha) [reply views/view-aquilon-services.php:31]: https://github.com/stfc/mimic/blob/main/inc/functions.inc.php#L28-L31

Comment (tofu-rocketry) [reply views/view-aquilon-services.php:31]: Thanks.

**Review by tofu-rocketry — APPROVED** [review]
Seems ok, though I'm not so hot on PHP. Maybe get a review from @gregcorbett ?

File: views/view-aquilon-services.php
Comment (gregcorbett) [inline views/view-aquilon-services.php:23]: an assignemnt in a while loop condition?

Comment (jrha) [reply views/view-aquilon-services.php:23]: yep, fairly standard DB access stuff, keep looping until we run out of rows.

**Review by gregcorbett — APPROVED** [review]
I mean, I didn't weep with eldritch madness while looking at the PHP code - so that's a win right?

**Pull request merged**
